packages/app-project/src/components/Announcements/components/FinishedAnnouncement/FinishedAnnouncementConnector.spec.js

```diff
@@ -7,7 +7,7 @@ describe('Component > FinishedAnnouncementConnector', function () {
 
   it('should show a results link if results page exists', function () {
     render(<DefaultStory />)
-    const link = screen.findByLabelText('Announcements.FinishedAnnouncement.seeResults')
-    expect(link).exists()
+    const link = screen.getByRole('link')
+    expect(link?.getAttribute('href')).to.equal('/projects/zookeeper/galaxy-zoo/about/results')
```

Comment:
The tests don't use `next.config.js`, which is why the app base path isn't included in these link URLs.